### PR TITLE
Unification of checks

### PR DESCRIPTION
**Pull request by Guimove**
- [x] Replace some bash function by their python brother 
- [x] Unification of some checks between Linux en Mac

As much as possible !

### CODE REVIEW

File: centralreport/cr/collectors.py [hunk 1]
```diff
@@ -132,65 +166,27 @@ def get_memory(self):
...
         """
-            Gets the host load average.
+            Gets load average.
         """
 
-        dict_iostat = self.get_IO_stats()
+        loadavg_result = os.getloadavg()
+
+        # On va spliter en fonction des espaces
+        dict_loadavg = loadavg_result
```
Comment (miniche) [inline centralreport/cr/collectors.py]: I think this line is not really useful, no?

**Comment by miniche** [comment]
Another possible improvement, in `Debian Collector > get_infos`, lines 280 to 291:

I think you can remplace this (my? :grin:) spaghetti code with only the following command:

``` bash
cat /etc/lsb-release | grep "DISTRIB_RELEASE" | awk 'BEGIN { FS="=" } END { print $2; }'
```

Yeah, I know, you love this command :kissing_heart:
Tested with success on Ubuntu 11.10.

**Comment by Guimove** [comment]
This only work on Ubuntu, /etc/lsb-release doesn't exist on Debian.
But I'll use 

```
platform.linux_distribution()
```

This returns : 

```
('debian', '6.0.7', '')
```

**Comment by miniche** [comment]
Yeah, I know my solution only works on Ubuntu, but the test for Debian has already been done 5 lines before.

And, very good idea for your improvement! I'm impressed :stuck_out_tongue_winking_eye:

**Comment by miniche** [comment]
Seems good for me :+1:

**Pull request merged**